backend/data/src/main/kotlin/io/tolgee/configuration/tolgee/machineTranslation/AwsMachineTranslationProperties.kt

```diff
@@ -6,6 +6,7 @@ import org.springframework.boot.context.properties.ConfigurationProperties
 open class AwsMachineTranslationProperties(
   override var defaultEnabled: Boolean = true,
   override var defaultPrimary: Boolean = false,
+  var enabled: Boolean = false,
```

Comment:
This is regrettable to change the config file, but now that `accessKey` and `secretKey` are no longer required to be specified, we needed a way to show that this was enabled and a client should be created. Are we able to use `defaultEnabled` instead for this purpose?

Reply:
`defaultEnabled` has another purpose. When `true` this translation provider is enabled by default, for all the projects.

Reply:
Let's make the property nullable. If it's null it's going to be enabled when access and secret keys are both set. This way it's not breaking change.